### ISSUE TITLE
Guilty Gear Xrd Rev2's wrong pixel ratio

### ISSUE THREAD

**Issue by Fowz** [Codeusa/Borderless-Gaming#384]
So I'm searching for an answer for this bug forever, some people say that they don't have any problem but I just can't make it work, even with this solution: https://github.com/Codeusa/Borderless-Gaming/issues/314

To sum it up:
![image](https://user-images.githubusercontent.com/10855150/37198217-5b663310-235c-11e8-9689-96a7bd7bfd94.png)
![image](https://user-images.githubusercontent.com/10855150/37198243-73229480-235c-11e8-91e5-2821656e53b8.png)

Basically Guilty Gear Xrd is gaining more 25px on height, while even if I offset it by the same amout it still gets 745px of height.
I don't know if the game being on Unreal 3 or something should give these problems, well, here it is, any workaround is welcome.

- AMD Athlon II X4 651K
- XFX Radeon HD7770 1Gb GDDR5
- 8Gb DDR3 1333mhz RAM
- 1366x768@60hz display
- Windows 7 Ultimate x64
